stat20-r-packages.r

```diff
@@ -23,7 +23,8 @@ set_libs <- c(
   "openintro", "2.4.0",
   "janitor", "2.2.0",
   "quarto", "1.2",
-  "fs", "1.6.3"
+  "fs", "1.6.3",
+  "vcd", "1.4-12"
```

Comment:
should this be `1.4.12`?

Reply:
And I looked at some other packages to see what they looked like and they are all of the form n.k1.k2

Reply:
Hmm, well lets just test locally if the hyphen works. one sec.